go.mod

```diff
@@ -1,12 +1,19 @@
 module github.com/datastax/go-cassandra-native-protocol
 
-go 1.16
+go 1.17
```

Comment:
I'm changing to 1.17 just because the Makefile requires 1.17 (to use `go install`). Note that this has little to no impact on users of this library as it remains backwards-compatible as long as we don't use any language feature specific to 1.17 (which we don't).